filewatcher/filewatcher_test.go

```diff
@@ -20,6 +20,25 @@ func parser(f io.Reader) (interface{}, error) {
 	return io.ReadAll(f)
 }
 
+func overwriteFile(tb testing.TB, path string, content io.Reader) {
```

Comment:
🔕  might be good to add a comment to this explaining why we have it, at least for future readers learning?

Reply:
Done.